examples/response-with-multiparts/response_with_multiparts.bal

```diff
@@ -119,7 +119,22 @@ function handleContent(mime:Entity bodyPart) {
         } else {
             log:printError("Error in parsing text data", err = payload);
         }
-    }
+    } else if (mime:APPLICATION_PDF == baseType) {
+        // [Extracts byte stream](https://ballerina.io/learn/api-docs/ballerina/#/ballerina/http/latest/http/classes/Response#getByteStream) from the body part and save it as a file.
+        var payload = bodyPart.getByteStream();
+        if (payload is stream<byte[], io:Error>) {
+            //Writes the incoming stream to a file using `io:fileWriteBlocksFromStream` API by providing the file location which the content should be written to.
+            io:Error? result = io:fileWriteBlocksFromStream(
+                                    "./files/ReceivedFile.pdf", streamer);
```

Comment:
It will be more readable if we take this to one line.

Reply:
Same reason applicable here as well